### ISSUE TITLE
Not enough spacing when using >10 core processor

### ISSUE THREAD

**Issue by Nightliss** [raiguard/ModernGadgets#265]
Hey rai,

thanks for updating the suite for HWInfo v7 :)

I've got an issue with spacing with core name, temp and voltage:
![image](https://user-images.githubusercontent.com/30467858/140563219-3e21e3d2-6de0-46b3-9b82-dcb18fd2ac61.png)

As I imagine making the widget wider would not look good so maybe is there a way to parse the values to drop decimal separator?

**Comment by SilverAzide** [comment]
There was a recent change to HWiNFO in which it started reporting temps to one decimal place for Ryzen processors (Intel is still no decimals).  It is -- unfortunately -- not a simple fix.

**Comment by raiguard** [comment]
In the meantime, you could disable core voltages or something. Alternatively, you can increase the gadget width by setting `bgWidth=200` (default is 150) in `GlobalSettings.inc`.

**Comment by Nightliss** [comment]
Not that I didn't want that, just thought it would not scale well with wider container. I didn't found the variable you have mentioned in that file, but I did find `gadgetBgWidth` in `StyleSheet.inc`. Changing it to 165 make it look great. It also changes other gadgets' width so it matches :) Thanks!

![image](https://user-images.githubusercontent.com/30467858/140576580-2148f182-a439-445c-84ac-6a26877f026c.png)

**Comment by raiguard** [comment]
Yeah, `bgWidth` does not exist in `GlobalSettings`, but adding it there will override what is in `StyleSheet`, and will be preserved across versions.

**Comment by Y0dle** [comment]
I noticed this issue as well, another workaround for now: you can change the number of decimals in HWiNFO's sensor settings.

**Comment by raiguard** [comment]
This will be addressed in the CPU meter rewrite to fix #278. I will be removing the "core" part of each label which will free up a ton of space for the stuff in the middle.
